src/server.ts

```diff
@@ -163,6 +175,7 @@ export class JSONRPCServer<ServerParams = void> {
     serverParams?: ServerParams
   ): Promise<JSONRPCResponse | null> {
     const method = this.nameToMethodDictionary[request.method];
+    const defaultMethod = this.defaultMethod;
```

Comment:
I think making it like this, we can avoid adding another `if` statement below:
```suggestion
    const method = this.nameToMethodDictionary[request.method] ?? this.defaultMethod;
```